internal/scripts/prune-preview-deploys.ts

```diff
@@ -163,13 +164,35 @@ async function listFlyioPreviewApps() {
 	return appNames.filter((name) => ZERO_CACHE_APP_REGEX.test(name))
 }
 
+async function listAmazonClusters() {
+	const client = new ECSClient({ region: 'eu-north-1', profile: 'preview' })
```

Comment:
i wonder if we might get into trouble with `profile: 'preview'` when running on CI like we did with the sst config, where we had to use `process.env.CI` to make it undefined to prevent aws complaining about not being able to find the config for the profile?

Reply:
Yeah, you were right. Remove the profile now.